sylt-parser/src/statement.rs

```diff
@@ -302,6 +311,84 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
             (ctx, Use { path: path_ident, name: alias, file })
         }
 
+        // `from path/to/file use var`
+        [T::From, ..] => {
+            let ctx = ctx.skip(1);
+            let (ctx, path_ident) = path(ctx)?;
+            let path = &path_ident.name;
+            let name = path
+                .trim_start_matches("/")
+                .trim_end_matches("/")
+                .to_string();
+            let file = {
+                let parent = if path.starts_with("/") {
+                    ctx.root
+                } else {
+                    ctx.file.parent().unwrap()
+                };
+                // Importing a folder is the same as importing exports.sy
+                // in the folder.
+                parent.join(if path == "/" {
+                    format!("exports.sy")
+                } else if path_ident.name.ends_with("/") {
+                    format!("{}/exports.sy", name)
+                } else {
+                    format!("{}.sy", name)
+                })
+            };
+            let mut ctx = expect!(ctx, T::Use, "Expected 'use' after path");
+            let mut imports = Vec::new();
+
+            let paren = matches!(ctx.token(), T::LeftParen);
+            let (ctx_, skip_paren) = if paren {
+                ctx = ctx.skip(1);
+                ctx.push_skip_newlines(true)
+            } else {
+                ctx.push_skip_newlines(false)
+            };
+            ctx = ctx_;
+            loop {
+                match ctx.token() {
+                    T::RightParen | T::Newline => break,
+                    T::Identifier(name) => {
+                        let ident = Identifier { name: name.clone(), span: ctx.span() };
+                        ctx = ctx.skip(1);
+                        let alias = if matches!(ctx.token(), T::As) {
+                            ctx = ctx.skip(1);
+                            let _alias = match ctx.token() {
+                                T::Identifier(name) => {
+                                    Some(Identifier { name: name.clone(), span: ctx.span() })
+                                }
+                                _ => raise_syntax_error!(ctx, "Expected identifier after 'as'"),
+                            };
+                            ctx = ctx.skip(1);
+                            _alias
+                        } else {
+                            None
+                        };
```

Comment:
Maybe this should be broken out - so it can be used in the other import aswell? It's gonna be a lot easier to keep this consistent if it's the same parsing function.